From 80dac47ee3aa26290cda962ab2531afb054bf47a Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Thu, 3 Jun 2021 00:54:30 -0700
Subject: [PATCH] [HACK] Disable colorized output in M-l window list

vid_attr is not supported by NetBSD's libcurses, and this is not
crucial feature.
---
 ui.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ui.c b/ui.c
index 64e83d3..407672c 100644
--- a/ui.c
+++ b/ui.c
@@ -686,7 +686,6 @@ static void windowList(const struct Window *window) {
 
 		char buf[TimeCap];
 		strftime(buf, sizeof(buf), uiTime.format, localtime(&line->time));
-		vid_attr(colorAttr(Colors[Gray]), colorPair(Colors[Gray], -1), NULL);
 		printf("%s ", buf);
 
 		bool align = false;
@@ -702,7 +701,6 @@ static void windowList(const struct Window *window) {
 			size_t tab = strcspn(str, "\t");
 			if (tab < len) len = tab;
 
-			vid_attr(styleAttr(style), stylePair(style), NULL);
 			printf("%.*s", (int)len, str);
 			str += len;
 		}
-- 
2.32.0

